bootstrap/sql/org.postgresql.Driver/v008__create_db_connection_info.sql

```diff
@@ -66,5 +66,4 @@ jsonb_build_object('connection',jsonb_build_object(
 'password',json #>'{connection,config,password}',
 'provider',json #>'{connection,config,provider}'
 )), true)
-where servicetype = 'Superset';
-
+where servicetype = 'Superset';
```

Comment:
Again, this change needs to be undone. Otherwise we will have upgrade failures.

Reply:
my IntelliJ, removed the change